processing/src/main/java/io/druid/segment/IndexMerger.java

```diff
@@ -399,6 +416,35 @@ public String apply(@Nullable String input)
     return makeIndexFiles(indexes, outDir, progress, mergedDimensions, mergedMetrics, rowMergerFn, indexSpec);
   }
 
+  public static File reprocess(final File inDir, final File outDir, final IndexSpec indexSpec) throws IOException
+  {
+    return reprocess(inDir, outDir, new BaseProgressIndicator(), indexSpec);
+  }
+
+  public static File reprocess(
```

Comment:
if we add the progress parameter, it should be add the end, to make the order consistent with the method without the progress parameter